src/lib/market-service/market-service-manager.ts

```diff
@@ -93,7 +94,29 @@ export class MarketServiceManager {
         // Swallow error, not every asset will be with every provider.
       }
     }
-    if (!result) return null
+
+    // If we don't find any results, then we look for related assets
+    if (!result) {
+      const relatedAssetIds = await _getRelatedAssetIds(assetId)
+      if (!relatedAssetIds.length) return null
+
+      for (const relatedAssetId of relatedAssetIds) {
+        // Loop through market providers and look for related asset market data.
+        for (let i = 0; i < this.marketProviders.length && !result; i++) {
+          try {
+            result = await this.marketProviders[i].findByAssetId({ assetId: relatedAssetId })
```

Comment:
thinking out loud we may want to restrict these calls to return only the price and `changePercent24Hr` as a subset, not the full thing?